### DISCUSSION TITLE
Questions about cog annotation:

### DISCUSSION THREAD

**Discussion by fersity** [oschwengers/bakta#352] [Q&A]
When annotating a draft genome (contig) of an uncommon Marine bacterium, I found that the annotated COG information was less than 200, while the CDS had more than 3500 (most of the COG map generated by bakta_plot was gray).  In addition, a previous attempt to annotates the complete E. coli genome yielded a wealth of COG annotations (circos COG diagram is in beautiful color).
What is the reason, please?
Can I incorporate external COG annotation information into the json file resulting from the annotation? Thank you.

**Comment by oschwengers** [comment]
Hi and thanks for asking. There is a simple answer to this: In Ecoli, you find many genes which are annotated in COG. In less well studies species like your marine bacterium this is not the case.

However, I just recently recognized that there is a 2024 update of the COG database. One could use that information to improve the COG coverage of these less well studies species. We're keen to look into this for an upcoming database update.
